lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -5035,11 +5035,23 @@ sub insertFrontMatter {
       local $LaTeXML::BOX = Box('', $STATE->lookupValue('font'), '', T_SPACE);
       foreach my $item (@$list) {
         my ($tag, $attr, @stuff) = @$item;
+        # add a dedicated class for frontmatter notes,
+        # in the case we want to style those uniformly.
+        if ($tag eq 'ltx:note') {
+          $attr ||= {};
+          $$attr{class} = ($$attr{class} ? $$attr{class} . ' ' : '') . 'ltx_note_frontmatter'; }
```

Comment:
Since everything seen here is frontmatter of some sort, I'm wondering if this should just add `ltx_frontmatter`, or possibly `$tag . '_frontmatter'` (s/:/_/g) ?

Reply:
Alright, updated to a generic `ltx_frontmatter` class, which sadly also means I updated tens of tests in the process. 

The styling I'm interested in can now be done via a CSS selector on `.ltx_note.ltx_frontmatter`, which is perfectly usable.

Reply:
Ugh. and having that class on `ltx:title`, `ltx:creator`, etc is annoyingly pointless. Maybe you were right the first time?

Reply:
Luckily reverting is much, much faster :> I've rewinded things back.